gensim/models/doc2vec.py

```diff
@@ -1505,7 +1505,7 @@ def __iter__(self):
             fname = os.path.join(self.dirname, fname)
             if not os.path.isfile(fname):
                 continue
-            for item_no, line in enumerate(utils.smart_open(fname)):
+            for item_no, line in enumerate(utils.open(fname, 'rb')):
```

Comment:
Let's be consistent:

```python
# importing
from gensim import utils

# opening
with utils.open(...) as fin:
    ...
```

Here you've done "from gensim.utils import open" but instead of using it, you've used "utils.open" instead. That's a bit confusing.

Reply:
no problem, I'll change all occurrences and update shortly

Reply:
@mpenkov done. Can you please review again?

Reply:
Looking good. Can you please wrap the open in a context manager?

Reply:
done